channel/state/outcome/outcome.go

```diff
@@ -79,6 +90,27 @@ func (allocations Allocations) Affords(given Allocation, funding *big.Int) bool
 	return false
 }
 
+// DepositSafetyThreshold returns the amount of this asset that a user with
+// the specified interests must see on-chain before the safe recoverability of
+// their own deposts is guaranteed
+func (s SingleAssetExit) DepositSafetyThreshold(interests []types.Destination) *big.Int {
```

Comment:
I tried that out over on #90 . I have a feeling you may not be in favour, since that splits up the methods that define a "class" across several files. But take a look, anyway.

Reply:
As far as I can tell it mostly seems to do the opposite - each file is named for the type it contains, and attached methods live in the file. I'll see what happens when trying to merge this after some other changes have been made, but I expect the path-of-least-resistance will be to chop it up again inside this branch.

Reply:
Yes it mostly does that, but not without exception -- it splits off some of the methods for `SingleAssetExit` and for `Exit`(exactly those that are to do with safe deposit thresholds) and houses them in a dedicated file named for that purpose. 

Let's focus on this PR for now, and tackle this question post-merge?

Reply:
:+1: